GOVERNANCE.md

```diff
@@ -0,0 +1,124 @@
+> **Note:**
+>
+> This is a **Work in Progress**. Stay tuned for more follow-up updates!
+
+# TiDB Governance
+
+This document describes the governance rules of the TiDB project (organization). It is meant to be followed by all the repositories in the project and the TiDB community.
+
+## Principles
+
+The TiDB community adheres to the following principles:
+
+- **Open**: TiKV is open source, as advertised on our [website](https://pingcap.com) or the [PingCAP Organization](https://github.com/pingcap) on github.
+- **Welcoming and respectful**: See Code of Conduct below.
+- **Transparent and accessible**: Work and collaboration are done in public.
+- **Merit**: Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+
+## Code of Conduct
+
+The TiDB community follows the [TiDB Code of Conduct](/CODE_OF_CONDUCT.md). Here are some excerpts:
+
+> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Community Roles and Responsibilities
+
+See <https://github.com/pingcap/community/tree/master/architecture> for details.
+
+## Community Structure
```

Comment:
Is it included in the **TiDB Community Architecture**?

Reply:
Maybe we can call it Community Groups? We just want to list the groups that are closely related to Governance, not introduce the structure.

Reply:
OK. But seems it is duplicated with the group descriptions in TiDB Community Architecture?

Reply:
It is. Actually, it's intended here with a focus on governance.  In TiKV, we only have a membership.md that focuses on the Roles. So it makes sense to introduce the overall groups here because these groups are actual governing groups. 
Maybe we can consider updating the architecture?